pkg/rpc/server.go

```diff
@@ -181,9 +181,17 @@ Methods:
 
 		results = peers
 
-	case "validateaddress", "getblocksysfee", "getcontractstate", "getrawmempool", "getrawtransaction", "getstorage", "submitblock", "gettxout", "invoke", "invokefunction", "invokescript", "sendrawtransaction":
+	case "getblocksysfee", "getcontractstate", "getrawmempool", "getrawtransaction", "getstorage", "submitblock", "gettxout", "invoke", "invokefunction", "invokescript", "sendrawtransaction":
 		results = "TODO"
 
+	case "validateaddress":
+		param, exists := reqParams.ValueAt(0)
+		if !exists {
+			resultsErr = newError(-2146233086, http.StatusOK, "Index was out of range. Must be non-negative and less", "", nil)
```

Comment:
As @aprasolova suggested at https://github.com/CityOfZion/neo-go/pull/131 maybe we need constant for magic error code?

Reply:
@AlexVanin needs merge #131 before... than I remove this

Reply:
@im-kulikov no problem, I hope it will be merged soon

Reply:
I already fix that in #131